presentation/timer/src/main/java/com/together/study/timer/service/TimerWorkingService.kt

```
@@ -0,0 +1,126 @@
+                    startTicking()
+                }
+        }
+    }
+
+    fun stop() {
+        val id = timerId ?: return
+
+        scope.launch {
+            try {
+                stopTimerUseCase(id)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)